doc/nrf/libraries/nrf_security/doc/backend_config.rst

```diff
@@ -33,16 +33,6 @@ Oberon backend
 
 Setting the Kconfig option :kconfig:option:`CONFIG_CC3XX_OBERON` enables legacy crypto support using :ref:`nrf_oberon_readme`.
 
-
-.. _nrf_security_backends_orig_mbedtls:
```

Comment:
You have a real doc failure due to the declaration of this label disappearing, but there still being a reference to it in knownissues.rst

/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/known_issues.rst:544:undefined label: 'nrf_security_backends_orig_mbedtls'

You need to delete the reference I suppose.